antarest/study/service.py

```diff
@@ -1352,7 +1352,7 @@ def import_study(
             public_mode=PublicMode.NONE if group_ids else PublicMode.READ,
             groups=group_ids,
         )
-        study = self.storage_service.raw_study_service.import_study(study, stream)
+        study = cast(RawStudy, self.storage_service.raw_study_service.import_study(study, stream))
```

Comment:
Here we should not need to cast, the method `import_study` should return a `RawStudy` instead